commands/host.vegman

```diff
@@ -257,8 +261,10 @@ function cmd_virtualmedia_mount {
     /xyz/openbmc_project/VirtualMedia/Legacy/USB1 \
     xyz.openbmc_project.VirtualMedia.Legacy \
     Mount sbyv "$imagefile" $readwrite ${type} i ${cred_pipe} > /dev/null  && \
-  echo "Virtual Media (USB1) mounted" || \
-  echo "Failed to mount Virtual Media (USB1)"
+  echo "Virtual Media (USB1) mounted" && \
+  log_audit "Host virtual media mount : ${imagefile}" || \
+  echo "Failed to mount Virtual Media (USB1)" && \
+  log_audit "Failed to mount Virtual Media (USB1)"
```

Comment:
I don't think we need to log failed mounts to audit log.
Also, that won't do what you expect it to, need parentheses, like this:

```
<mount_command> && \
(echo "... mounted" ; log_audit "Mounted VM (USB1) from ${imagefile}") || \
echo "Failed... "
```